CHANGES/1144.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed the library building in debug mode.
```

Comment:
(grammar)
```suggestion
Fixed building the library in debug mode.
```

Also, does do we think this is user-facing enough to get a `bugfix` note? If it's downstream, I'd go for `packaging`. And maybe symlink to `contrib`.

cc @bdraco what's your opinion here?

Reply:
I think it is a bug